Sources/Interaction/Manipulators/MouseRangeManipulator/index.js

```diff
@@ -112,6 +112,14 @@ function vtkMouseRangeManipulator(publicAPI, model) {
   //-------------------------------------------------------------------------
   publicAPI.onButtonDown = (interactor, renderer, position) => {
     model.previousPosition = position;
+    const glRenderWindow = interactor.getView();
+    // Ratio is the dom size vs renderwindow size
+    const ratio =
+      glRenderWindow.getContainerSize()[0] / glRenderWindow.getSize()[0];
+    // Get proper pixel range used by viewport in rw size space
+    const size = glRenderWindow.getViewportSize(renderer);
+    // rescale size to match mouse event position
+    model.size = size.map((v) => v * ratio);
```

Comment:
`model.containerSize` to be more explicit? `size` itself won't mean much in for a `MouseRangeManipulator`... but it's not exposed so maybe that's fine.

Reply:
Sure what ever